azure-monitor-opentelemetry/azure/monitor/opentelemetry/util/configurations.py

```diff
@@ -15,7 +61,121 @@ def _get_configurations(**kwargs) -> Dict[str, ConfigurationValue]:
+
+
+def _default_disable_metrics(configurations):
+    if DISABLE_METRICS_ARG not in configurations:
+        default = False
+        if DISABLE_METRICS_ENV_VAR in environ:
+            default = bool(environ[DISABLE_METRICS_ENV_VAR])
+        configurations[DISABLE_METRICS_ARG] = default
+
+
+def _default_disable_tracing(configurations):
+    if DISABLE_TRACING_ARG not in configurations:
+        default = False
+        if DISABLE_TRACING_ENV_VAR in environ:
+            default = bool(environ[DISABLE_TRACING_ENV_VAR])
+        configurations[DISABLE_TRACING_ARG] = default
+
+
+def _default_logging_level(configurations):
+    if LOGGING_LEVEL_ARG not in configurations:
+        default = NOTSET
+        if LOGGING_LEVEL_ENV_VAR in environ:
+            default = int(environ[LOGGING_LEVEL_ENV_VAR])
+            # TODO: Match OTel env var usage when it is determined.
+        configurations[LOGGING_LEVEL_ARG] = default
+
+
+def _default_logger_name(configurations):
+    if LOGGER_NAME_ARG not in configurations:
+        default = ""
+        if LOGGER_NAME_ENV_VAR in environ:
+            default = environ[LOGGER_NAME_ENV_VAR]
+        configurations[LOGGER_NAME_ARG] = default
+
+
+def _default_logging_export_interval_ms(configurations):
+    if LOGGING_EXPORT_INTERVAL_MS_ARG not in configurations:
+        default = 5000
+        if LOGGING_EXPORT_INTERVAL_MS_ENV_VAR in environ:
+            default = int(environ[LOGGING_EXPORT_INTERVAL_MS_ENV_VAR])
+        configurations[LOGGING_EXPORT_INTERVAL_MS_ARG] = default
+
+
+# TODO: Design metric readers env var usage
+def _default_metric_readers(configurations):
+    if METRIC_READERS_ARG not in configurations:
+        default = []
+        if _METRIC_READERS_ENV_VAR in environ:
+            default = loads(environ[_METRIC_READERS_ENV_VAR])
+        configurations[METRIC_READERS_ARG] = default
+
+
+# TODO: Design views env var usage
+def _default_views(configurations):
+    if VIEWS_ARG not in configurations:
+        # TODO tuple or list
+        default = []
+        if _VIEWS_ENV_VAR in environ:
+            default = loads(environ[_VIEWS_ENV_VAR])
+        configurations[VIEWS_ARG] = default
+
+
+# TODO: remove when sampler uses env var instead
+def _default_sampling_ratio(configurations):
+    if SAMPLING_RATIO_ARG not in configurations:
+        default = 1.0
+        if SAMPLING_RATIO_ENV_VAR in environ:
+            default = float(environ[SAMPLING_RATIO_ENV_VAR])
+        configurations[SAMPLING_RATIO_ARG] = default
+
+
+def _default_tracing_export_interval_ms(configurations):
+    if TRACING_EXPORT_INTERVAL_MS_ARG not in configurations:
+        configurations[TRACING_EXPORT_INTERVAL_MS_ARG] = None
+
+
+def _default_instrumentation_config(configurations):
+    if INSTRUMENTATION_CONFIG_ARG not in configurations:
+        default = {}
+        if INSTRUMENTATION_CONFIG_ENV_VAR in environ:
+            default = loads(environ[INSTRUMENTATION_CONFIG_ENV_VAR])
```

Comment:
We probably should have some error handling for `json.loads` in case it is not able to parse the json.